setup.py

```diff
@@ -79,6 +79,34 @@ def merge_req_lists(*req_lists: List[str]) -> List[str]:
     "pytelegrambotapi",
 ]
 
+clickhouse_dependencies = ["aiochclient>=2.2.0", "httpx<=0.23.0", "sqlparse==0.4.4"]
+
+requests_requirements = [
```

Comment:
We already have that object in this file.

Reply:
Moved to test dependencies

Reply:
There are still two objects named `request_requirements` in `setup.py`.